### DISCUSSION TITLE
I need help i played a rank role game and it didnt count both for my mmr and cavern crawl

### DISCUSSION THREAD

**Discussion by Kirbyracer** [ValveSoftware/Dota2-Gameplay#2928] [Q&A]
Rank role game didnt count because game servers are busy. What do i do its already more than 14 hours since that game it didnt reflect on my mmr and cavern crawl

**Comment by DavinS1** [comment]
Pleased to see your issue as I think I have the same problem, I am also confused and in need of light on this same issue.


[scoresense.com
](https://www.scoresense.me/)
